beangulp/petl_utils.py

```diff
@@ -1,11 +1,11 @@
 """Utilities using petl.
 """
 
-from typing import Optional
+from typing import Optional, Set
 import datetime
 import re
 
-import petl
+import petl  # type: ignore
```

Comment:
Why this ignore?

Reply:
didn't look much further into it - is this file even supposed to be part of the package? petl doesn't seem to be listed in the dependencies